bin/spec-configs/accel_v14/runOne

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 set -x
-export AOMP=~/rocm//aomp_16.0-40
-export AOMP_GPU=gfx90a
+
+: {AOMP:='~/rocm/aomp'}
```

Comment:
thse might be more portable  for accel.... :
export AOMP=${AOMP:-`ls -d /opt/rocm-*/llvm |sort|tail -1`}
export GFX=${GFX:-`rocm_agent_enumerator | grep -v gfx000|uniq`}

Reply:
Thanks! Updated the patch